### PR TITLE
Create oki.js

### PR DESCRIPTION
**Pull request by jn-lo**
alert js

### CODE REVIEW

File: oki.js [hunk 1]
```diff
@@ -0,0 +1 @@
+alert("good morning")
```
Comment (jhnn-you) [inline oki.js:1]: bakhalti wachno hada

Comment (jn-lo) [reply oki.js:1]: haka brk

Comment (jn-lo) [reply oki.js:1]: zatha man 3andi

Comment (jhnn-you) [reply oki.js:1]: kifeh hadi twzid man 3andek

Comment (jn-lo) [reply oki.js:1]: haka brk

**Pull request merged**